### ISSUE TITLE
Cannot read property 'validate' of undefined

### ISSUE THREAD

**Issue by deyhle** [raml2html/raml2html#362]
There seems to be a undocumented breaking change in the current version. While 6.0.0 worked, it now fails with the following error:

```
TypeError: Cannot read property 'validate' of undefined
    at Object.render (/usr/src/app/node_modules/raml2html/index.js:33:40)
```

I'm calling `raml2html.render(object, config)` as told in the docs. Apparently there is now a third parameter that needs to be set. Such a change cannot come in a non-major release without breaking stuff.

**Comment by kevinrenskers** [comment]
Sorry about that! 6.4.1 just got published with a fix. Could you please confirm if everything works as expected again?

**Comment by deyhle** [comment]
Thanks for the quick fix, it works again with 6.4.1!
